connector-servicenow/src/main/java/com/redhat/cloud/notifications/connector/servicenow/ServiceNowRouteBuilder.java

```diff
@@ -0,0 +1,60 @@
+package com.redhat.cloud.notifications.connector.servicenow;
+
+import com.redhat.cloud.notifications.connector.ConnectorConfig;
+import com.redhat.cloud.notifications.connector.EngineToConnectorRouteBuilder;
+import org.apache.http.conn.ssl.NoopHostnameVerifier;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import static com.redhat.cloud.notifications.connector.ConnectorToEngineRouteBuilder.SUCCESS;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.ID;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.ORG_ID;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.TARGET_URL;
+import static com.redhat.cloud.notifications.connector.servicenow.ExchangeProperty.ACCOUNT_ID;
+import static com.redhat.cloud.notifications.connector.servicenow.ExchangeProperty.TARGET_URL_NO_SCHEME;
+import static com.redhat.cloud.notifications.connector.servicenow.ExchangeProperty.TRUST_ALL;
+import static com.redhat.cloud.notifications.connector.servicenow.ServiceNowTrustAllManager.getSslContextParameters;
+import static org.apache.camel.LoggingLevel.INFO;
+import static org.apache.camel.builder.endpoint.dsl.HttpEndpointBuilderFactory.HttpEndpointBuilder;
+
+@ApplicationScoped
+public class ServiceNowRouteBuilder extends EngineToConnectorRouteBuilder {
+
+    @Inject
+    ConnectorConfig connectorConfig;
+
+    @Inject
+    MigrationFilter migrationFilter;
+
+    @Override
+    public void configureRoute() {
+        from(direct(ENGINE_TO_CONNECTOR))
+                .routeId(connectorConfig.getConnectorName())
+                // TODO For migration purposes - Remove ASAP!
+                .filter(migrationFilter)
+                .process(new BasicAuthenticationProcessor())
+                // SSL certificates may or may not be verified depending on the integration settings.
+                .choice()
+                .when(exchangeProperty(TRUST_ALL))
+                        .toD(buildServiceNowEndpoint(true), connectorConfig.getEndpointCacheMaxSize())
+                .endChoice()
+                .otherwise()
+                        .toD(buildServiceNowEndpoint(false), connectorConfig.getEndpointCacheMaxSize())
+                .end()
+                .log(INFO, getClass().getName(), "Delivered event ${exchangeProperty." + ID + "} " +
+                        "(orgId ${exchangeProperty." + ORG_ID + "} account ${exchangeProperty." + ACCOUNT_ID + "}) " +
+                        "to ${exchangeProperty." + TARGET_URL + "}")
+                .to(direct(SUCCESS));
```

Comment:
I'll ignore but not because I want to 😄 This message comes from the eventing app and I know there are Kibana dashboards built on top of it. I didn't modify the old message to avoid breaking the dashboards, for now. We'll certainly change the messages later.

Reply:
Perfect, thanks!